app/src/main/java/com/orange/ouds/app/ui/tokens/TokenProperty.kt

```diff
@@ -35,407 +38,256 @@ sealed class TokenProperty<T>(
-        OudsSizeKeyToken.Icon.WithBody.Large.SizeSmall -> 70
-        OudsSizeKeyToken.Icon.WithBody.Large.SizeMedium -> 71
-        OudsSizeKeyToken.Icon.WithBody.Large.SizeLarge -> 72
-        OudsSizeKeyToken.Icon.WithLabel.Small.SizeExtraSmall -> 80
-        OudsSizeKeyToken.Icon.WithLabel.Small.SizeSmall -> 81
-        OudsSizeKeyToken.Icon.WithLabel.Small.SizeMedium -> 82
-        OudsSizeKeyToken.Icon.WithLabel.Small.SizeLarge -> 83
-        OudsSizeKeyToken.Icon.WithLabel.Medium.SizeExtraSmall -> 90
-        OudsSizeKeyToken.Icon.WithLabel.Medium.SizeSmall -> 91
-        OudsSizeKeyToken.Icon.WithLabel.Medium.SizeMedium -> 92
-        OudsSizeKeyToken.Icon.WithLabel.Medium.SizeLarge -> 93
-        OudsSizeKeyToken.Icon.WithLabel.Large.SizeExtraSmall -> 100
-        OudsSizeKeyToken.Icon.WithLabel.Large.SizeSmall -> 101
-        OudsSizeKeyToken.Icon.WithLabel.Large.SizeMedium -> 102
-        OudsSizeKeyToken.Icon.WithLabel.Large.SizeLarge -> 103
-        OudsSizeKeyToken.Icon.WithLabel.Large.SizeExtraLarge -> 104
-        OudsSizeKeyToken.Icon.WithLabel.ExtraLarge.SizeSmall -> 110
-        OudsSizeKeyToken.Icon.WithLabel.ExtraLarge.SizeMedium -> 111
-        OudsSizeKeyToken.Icon.WithLabel.ExtraLarge.SizeLarge -> 112
-    }
-
-val OudsSpaceKeyToken.ColumnGap.order: Int
-    get() = when (this) {
-        OudsSpaceKeyToken.ColumnGap.None -> 0
-        OudsSpaceKeyToken.ColumnGap.Smash -> 1
-        OudsSpaceKeyToken.ColumnGap.Shortest -> 2
-        OudsSpaceKeyToken.ColumnGap.Shorter -> 3
-        OudsSpaceKeyToken.ColumnGap.Short -> 4
-        OudsSpaceKeyToken.ColumnGap.Medium -> 5
-        OudsSpaceKeyToken.ColumnGap.Tall -> 6
-        OudsSpaceKeyToken.ColumnGap.Taller -> 7
-        OudsSpaceKeyToken.ColumnGap.Tallest -> 8
-    }
-
-val OudsSpaceKeyToken.Fixed.order: Int
-    get() = when (this) {
-        OudsSpaceKeyToken.Fixed.None -> 0
-        OudsSpaceKeyToken.Fixed.Smash -> 1
-        OudsSpaceKeyToken.Fixed.Shortest -> 2
-        OudsSpaceKeyToken.Fixed.Shorter -> 3
-        OudsSpaceKeyToken.Fixed.Short -> 4
-        OudsSpaceKeyToken.Fixed.Medium -> 5
-        OudsSpaceKeyToken.Fixed.Tall -> 6
-        OudsSpaceKeyToken.Fixed.Taller -> 7
-        OudsSpaceKeyToken.Fixed.Tallest -> 8
-        OudsSpaceKeyToken.Fixed.Spacious -> 9
-        OudsSpaceKeyToken.Fixed.Huge -> 10
-        OudsSpaceKeyToken.Fixed.Jumbo -> 11
-    }
-
-val OudsSpaceKeyToken.Inset.order: Int
-    get() = when (this) {
-        OudsSpaceKeyToken.Inset.None -> 0
-        OudsSpaceKeyToken.Inset.Smash -> 1
-        OudsSpaceKeyToken.Inset.Shortest -> 2
-        OudsSpaceKeyToken.Inset.Shorter -> 3
-        OudsSpaceKeyToken.Inset.Short -> 4
-        OudsSpaceKeyToken.Inset.Medium -> 5
-        OudsSpaceKeyToken.Inset.Tall -> 6
-        OudsSpaceKeyToken.Inset.Taller -> 7
-        OudsSpaceKeyToken.Inset.Tallest -> 8
-        OudsSpaceKeyToken.Inset.Spacious -> 9
-    }
+    val path = parentClassName.removePrefix("Ouds")
```

Comment:
A detail but maybe we can factorize the prefix cause it is also used in `getTokens()` method.